<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LinuxProject1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>62</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>Bluetooth On</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>Bluetooth Off</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>Wifi On</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>Wifi Off</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>270</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="5" column="1">
      <widget class="QLCDNumber" name="lcdNumber_22"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Time Remaining:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcdNumber_25"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>V-out:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>A-out:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>P-out:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLCDNumber" name="lcdNumber_24"/>
     </item>
     <item row="6" column="1">
      <widget class="QLCDNumber" name="lcdNumber_23"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_22">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Output #2:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButton_7">
         <property name="text">
          <string>Output On</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_8">
         <property name="text">
          <string>Output Off</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string>Graph Tempurature</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_58">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Details:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>270</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="7" column="1">
      <widget class="QLCDNumber" name="lcdNumber_21"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Time Remaining:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>V-out:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_15">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Output #1:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcdNumber_18"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>P-out:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string>Graph Sample Input</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLCDNumber" name="lcdNumber_19"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>A-out:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLCDNumber" name="lcdNumber_20"/>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string>Output On</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_6">
         <property name="text">
          <string>Output Off</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_59">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Details:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>270</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcdNumber_44"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_42">
       <property name="text">
        <string>Time Remaining:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLCDNumber" name="lcdNumber_39"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_44">
       <property name="text">
        <string>A-out:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLCDNumber" name="lcdNumber_38"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_43">
       <property name="text">
        <string>V-out:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_45">
       <property name="text">
        <string>P-out:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLCDNumber" name="lcdNumber_43"/>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QRadioButton" name="radioButton_15">
         <property name="text">
          <string>Output On</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_16">
         <property name="text">
          <string>Output Off</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_46">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Output #5:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_16">
       <property name="text">
        <string>Set Schedule</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_55">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Details:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>857</x>
      <y>270</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="label_48">
       <property name="text">
        <string>Time Remaining:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcdNumber_48"/>
     </item>
     <item row="5" column="1">
      <widget class="QLCDNumber" name="lcdNumber_45"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_50">
       <property name="text">
        <string>A-out:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_51">
       <property name="text">
        <string>P-out:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_49">
       <property name="text">
        <string>V-out:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLCDNumber" name="lcdNumber_47"/>
     </item>
     <item row="6" column="1">
      <widget class="QLCDNumber" name="lcdNumber_46"/>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QRadioButton" name="radioButton_17">
         <property name="text">
          <string>Output On</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_18">
         <property name="text">
          <string>Output Off</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_52">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Output #6:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_17">
       <property name="text">
        <string>Set Schedule</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_57">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Details:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>270</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="6" column="1">
      <widget class="QLCDNumber" name="lcdNumber_35"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_39">
       <property name="text">
        <string>P-out:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_38">
       <property name="text">
        <string>A-out:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLCDNumber" name="lcdNumber_36"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>Time Remaining:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcdNumber_37"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_37">
       <property name="text">
        <string>V-out:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLCDNumber" name="lcdNumber_34"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_15">
       <property name="text">
        <string>Set Schedule</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QRadioButton" name="radioButton_13">
         <property name="text">
          <string>Output On</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_14">
         <property name="text">
          <string>Output Off</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_40">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Output #4:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_54">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Details:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>270</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="3" column="0">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>Time Remaining:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcdNumber_33"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>V-out:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLCDNumber" name="lcdNumber_30"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>A-out:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLCDNumber" name="lcdNumber_31"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>P-out:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLCDNumber" name="lcdNumber_32"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_14">
       <property name="text">
        <string>Set Schedule</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_34">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Output #3:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButton_11">
         <property name="text">
          <string>Output On</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_12">
         <property name="text">
          <string>Output Off</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_35">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Details:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_60">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Controller #1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_61">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Controller #2:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Temp:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_2"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QRadioButton" name="radioButton_21">
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_22">
         <property name="text">
          <string>F</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>211</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Temp:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QRadioButton" name="radioButton_19">
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_20">
         <property name="text">
          <string>F</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Ambient Temp:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdNumber_5"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Humidity(inside):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdNumber_6"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Humidity(outside):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lcdNumber_7"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Number of Battery Cycles:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lcdNumber_8"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>V-out:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcdNumber_42"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>A-out:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLCDNumber" name="lcdNumber_41"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>P-out:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLCDNumber" name="lcdNumber_40"/>
     </item>
     <item row="6" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Temp:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_9"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QRadioButton" name="radioButton_23">
           <property name="text">
            <string>C</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_24">
           <property name="text">
            <string>F</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Battery Percentage:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GPIO ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GPIO OFF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>C:\Users\Sammy\Desktop\QCustomPlot\qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>ButtonClicked()</slot>
 </slots>
</ui>
